### PR TITLE
FW campaign logs

### PR DESCRIPTION
**Pull request by Amazinite**
PRing this now to get feedback on what I've done while I finish the rest. 
- [x] FW Intro (draft)
- [ ] FW Intro (final)
_
- [ ] FW Start (draft)
- [ ] FW Start (final)
_
- [ ] FW Middle (draft)
- [ ] FW Middle (final)
_
- [ ] FW Checkmate (draft)
- [ ] FW Checkmate (final)
_
- [ ] FW Reconciliation (draft)
- [ ] FW Reconciliation (final)
_
- [ ] FW Sideplots (draft)
- [ ] FW Sideplots (final)
_
- [x] FW Epilogue (draft)
- [ ] FW Epilogue (final)

### CODE REVIEW

File: data/free worlds intro.txt [hunk 1]
```diff
@@ -222,6 +223,9 @@ mission "Pact Questioning"
 			label quit
 			`	"Understood," he says. "Thank you for your time." He makes no effort to detain you. As you walk away, you glance over your shoulder a few times to make sure you are not being watched or followed, but see no one.`
 				decline
+				
+	on decline
+		log `Was questioned by a Navy officer for my work with the Pact. He was strangely friendly, but a little off putting.`
```
Comment (Bladewood) [inline data/free worlds intro.txt]: `a little off putting.`
I have no idea what that means.

File: data/free worlds intro.txt [hunk 2]
```diff
@@ -444,13 +452,30 @@ mission "FW Recon 3B"
...
+		log `While doing a recon mission for the Free Worlds I was followed by a Navy gunboat. Working for the Free Worlds might turn into a dangerous job if tension keeps building.`
+		fail
+
+mission "logbook: evaded gunboat"
+	invisible
+	to offer
+		not "followed by gunboat"
+		has "FW Recon 3B: done"
+	on offer
+		log `Did more work for JJ, the militia officer who asked me to scan a Navy Cruiser. `
```
Comment (Bladewood) [inline data/free worlds intro.txt]: trailing space

**Comment by Bladewood** [comment]
Should ship names be capitalized?

I see "Navy Cruiser" and "Navy gunboat".

File: data/free worlds intro.txt [hunk 3]
```diff
@@ -546,6 +572,26 @@ mission "FW Katya 1"
 	on decline
 		event "FW Katya Alt 2: ready" 21
 
+mission "logbook: karate chop decline"
+	invisible
+	to offer
+		has "karate chop"
+		has "FW Katya 1: offered"
+	on offer
+		log `I karate chopped a militia officer who was holding a gun to the back of my head. Luckily the gun wasn't loaded, but the karate chop didn't work as expected. (I knew I should have studied karate while on New Boston...)`
```
Comment (Bladewood) [inline data/free worlds intro.txt]: "karate chop" appears a bit too often in this.

`(I knew I should have studied karate while on New Boston...)`
"while on" feels weird. Try something like "back on".

File: data/free worlds intro.txt [hunk 4]
```diff
@@ -546,6 +572,26 @@ mission "FW Katya 1"
...
+
+mission "logbook: no karate chop decline"
+	invisible
+	to offer
+		not "karate chop"
+		has "FW Katya 1: offered"
+		not "FW Katya 1: active"
+		not "FW Katya 1: done"
+	on offer
+		log `A militia officer tried to recruit me to help the Free Worlds by holding a gun to the back of my head! The Navy better do something about this before someone gets hurt.`
```
Comment (Bladewood) [inline data/free worlds intro.txt]: `The Navy better do something about this before someone gets hurt.`

The start of this feels weird.

**Comment by Amazinite** [comment]
I need to check on that one. I lowercased gunboat because it was lowercased in the mission, but I don't know if cruiser is also lowercased.

File: data/free worlds intro.txt (hunk 2 above)
Comment (Amazinite) [inline data/free worlds intro.txt]: Pretty sure I meant to write more there then took a break.

File: data/free worlds intro.txt (hunk 1 above)
Comment (Amazinite) [inline data/free worlds intro.txt]: [Learn](https://www.google.com/search?q=off+putting&oq=off+putting&aqs=chrome..69i57j0j69i61j0l3.1198j0j7&sourceid=chrome&ie=UTF-8)

File: data/free worlds intro.txt [hunk 5]
```diff
@@ -1049,6 +1117,7 @@ mission "FW Katya 5C"
 	
 	on complete
 		payment 100000
+		log `We brought Brother Greg from Harmony to Greenrock to find information on who could have done the bombing and where they would have tested it. I never would have guess that monks and pirates would have such a good relationship with eachother.`
```
Comment (Bladewood) [inline data/free worlds intro.txt]: I would suggest some kind of introduction explaining who Brother Greg is (a monk).
> We brought a monk called Brother Greg ...

File: data/free worlds intro.txt [hunk 6]
```diff
@@ -1139,6 +1208,12 @@ mission "FW Katya 7"
 					accept
 				`	"Sorry, I've got some other missions I need to run instead of continuing this wild goose chase."`
 					decline
+					
+	on accept
+		log `Katya, Eyes, and I have checked three worlds for far for signs of nuclear testing, but with no luck. Our next stop is Deep.`
+		
+	on decline
+		log `Katya, Eyes, and I have checked three worlds for far for signs of nuclear testing, but with no luck. I've had enough, so I am going my own way now.`
```
Comment (Bladewood) [inline data/free worlds intro.txt]: > I've had enough, so I am going my own way now.

It feels better to contract the `I am` to `I'm`.

File: data/free worlds intro.txt (hunk 1 above)
Comment (tehhowch) [inline data/free worlds intro.txt]: Needs a hyphen, I believe was the intent:  "off-putting"

File: data/free worlds intro.txt [hunk 7]
```diff
@@ -1537,9 +1613,13 @@ mission "FW Escort 3"
 		
 			`	"Glad to hear it," he says. "The crew of that freighter is depending on you. I know I don't have to tell you what will happen to them if the pirates board them before you get there."`
 				accept
+				
+	on decline
+		log `Did a few escort jobs for the Free Worlds and one of their officers, Tomek, asked me to do a special combat mission for him. Combat is something they should be asking their militia to do; they signed up for that, I didn't.`
```
Comment (Bladewood) [inline data/free worlds intro.txt]: You know, I've always been confused by this mission. Nothing about it feels more [combat-y] than the previous escort missions.

File: data/free worlds intro.txt (hunk 3 above)
Comment (tehhowch) [inline data/free worlds intro.txt]: or just 'on'

File: data/free worlds intro.txt (hunk 6 above)
Comment (Amazinite) [inline data/free worlds intro.txt]: Y'all love contractions

File: data/free worlds intro.txt [hunk 8]
```diff
@@ -1740,6 +1820,7 @@ mission "FW Bounty 3"
 	on complete
 		"assisted free worlds" ++
 		payment 300000
+		log `Killed a number of pirates for the Free Worlds, including an Osprey with some strange alien weapons. The officer I spoke to said he would tell the Free Worlds Council about me.`
```
Comment (Bladewood) [inline data/free worlds intro.txt]: It's not explicitly stated that the weapons are of an alien nature. It's just conjecture. (I mean, they are, obviously, but the Free Worlds personnel have no way of knowing for sure.)

I think we can get a nicer ending than just "telling the Council about me".

File: data/free worlds intro.txt (hunk 7 above)
Comment (Amazinite) [inline data/free worlds intro.txt]: I'd say before there were multiple ships defending the merchants, but here it's just you.

Comment (Bladewood) [inline data/free worlds intro.txt]: Fair enough.

File: data/free worlds intro.txt (hunk 6 above)
Comment (Pointedstick) [inline data/free worlds intro.txt]: I'm so curious to know what part of the world you live in where English contractions aren't much used!

File: data/free worlds intro.txt (hunk 5 above)
Comment (Amazinite) [inline data/free worlds intro.txt]: >Also, "eachother" isn't valid; it should be "each other". Listen to the spellchecker, crying out in pain.

I always make this mistake and I will never stop.

File: data/free worlds intro.txt (hunk 6 above)
Comment (Amazinite) [inline data/free worlds intro.txt]: They are used, but you people want to use them where ever possible!

**Comment by Amazinite** [comment]
Bounced to doing the Epilogue since there are only 4 missions there.

File: data/free worlds intro.txt [hunk 9]
```diff
@@ -1139,6 +1208,12 @@ mission "FW Katya 7"
 					accept
 				`	"Sorry, I've got some other missions I need to run instead of continuing this wild goose chase."`
 					decline
+					
+	on accept
+		log `Katya, Eyes, and I have checked three worlds for far for signs of nuclear testing, but with no luck. Our next stop is Deep.`
```
Comment (jafdy) [inline data/free worlds intro.txt]: >for far for signs

>so far for signs

File: data/free worlds intro.txt (hunk 6 above)
Comment (jafdy) [inline data/free worlds intro.txt]: It's like you're speaking. So not in the slightest bit formal.

**Comment by endless-sky** [comment]
I think I'd prefer to avoid personal pronouns entirely in the log entries.

Making the entries in the second person wouldn't make sense, because it's
supposed to be the captain's logbook. But making them in the first person
also feels wrong, because it either implies that the "I" who is speaking is
not the player, or it puts specific words and judgments in the player's
mouth (which the game generally avoids doing).

Also note that the logbook only needs to include key plot points; there
doesn't need to be something for every variant of every mission. As with
all the other game text, it needs to be concise and interesting enough that
the player will never feel like reading the log entries is a waste of time.

**Comment by Amazinite** [comment]
>Also note that the logbook only needs to include key plot points; there
doesn't need to be something for every variant of every mission.

So can the logbook missions I've been doing where the log changes depending on a decision made or if you did something related prior to the mission?

**Comment by endless-sky** [comment]
I'm thinking that maybe I should just write the Free Worlds entries myself, to give an example of the sort of style and frequency that I think the log entries should have. (And, to get a better sense myself of how easy it is to avoid personal pronouns and to get the sort of voice that I'd like the log to have.)

**Comment by Amazinite** [comment]
Alright, I'll close this and focus on other things then.

**Pull request closed without merging**